### ISSUE TITLE
Listing on FMI tools page

### ISSUE THREAD

**Issue by chrbertsch** [vectorgrp/sil-kit-fmu-importer#2]
It would be good to list the list sil-kit-fmu-importer  on the FMI Tools page https://fmi-standard.org/tools/, as described here: https://github.com/modelica/fmi-standard.org/blob/main/CONTRIBUTING.md#updating-the-tools-list

**Comment by DominikHerr** [comment]
Hello @chrbertsch, thank you for notifying us about the FMI Tools page and how to add the SIL Kit FMU Importer. We will submit a PR to be added to the list in the next days.

**Comment by chrbertsch** [comment]
> Hello @chrbertsch, thank you for notifying us about the FMI Tools page and how to add the SIL Kit FMU Importer. We will submit a PR to be added to the list in the next days.

Next week the FMI User meeting at the Modelica Conference 2023 and it would be great to have SIL-KIT on our list! Thanks!

(The FMI Project also recommend to provide "compatibility information" for your tool https://github.com/modelica/fmi-standard.org/blob/main/CONTRIBUTING.md#compatibility-information . This will increase confidence of users in your FMI implementation.)

**Comment by DominikHerr** [comment]
@chrbertsch: The FMU Importer was added to the FMI Tools page a while ago and there seems to be no problem with the information we provided. For reference, the commit was [f64b4d8 on modelica/fmi-standard.org](https://github.com/modelica/fmi-standard.org/commit/f64b4d846cccab8fd1c42a10a1b8982216827b84).
We will update the entry if need arises. For now, I will close this issue.
If you have any remarks or questions feel free to open a new issue.
